packages/teams-js/src/public/dialog.ts

```diff
@@ -21,7 +21,15 @@ import {
 import { runtime } from './runtime';
 
 /**
- * Namespace to interact with the dialog module-specific part of the SDK.
+ * This group of capabilities enables apps to show dialogs. There are two primary types of dialogs: URL-based dialogs and [Adaptive Card](https://learn.microsoft.com/adaptive-cards/) dialogs.
+ * Both types of dialogs are shown in a modal dialog on top of your app.
```

Comment:
Is this also true for mobile dialogs?

Reply:
Hmm... I think we're getting deep into UX terminology territory. The behavior on mobile is "modal" in the sense that the user cannot interact with the underlying app while the "dialog" is showing. However, the tricky bit really is that the UX isn't a dialog in the classical sense of the word (it's just another full screen page on top of the full screen app page).

Maybe I can sidestep whether "modal" is the right word here by instead changing it to describe the behavior. How does the updated version look?